packages/components/psammead-caption/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-caption",
-  "version": "0.3.3",
+  "version": "0.4.0",
```

Comment:
On second thoughts, this should be `0.3.4` as it doesn't affect the primary code

Reply:
Done